exampleSite/content/docs/guide/shortcodes/asciinema.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
For now, let's exclude it from the sidebar (similar to the jupyter). We'll need to think about better grouping for the sidebar for these shortcodes